### PR TITLE
Add One in the Chamber and The Hidden to private match playlists

### PR DESCRIPTION
**Pull request by x3Karma**

### CODE REVIEW

**Comment by iCrazyBlaze** [comment]
These are fun gamemodes, they fit in well alongside Gun Game and Infection. This PR would make them more popular and add more variety to the game

File: Northstar.Custom/keyvalues/playlists_v2.txt [hunk 1]
```diff
@@ -1,644 +1,777 @@
-playlists
```
Comment (GeckoEidechse) [inline Northstar.Custom/keyvalues/playlists_v2.txt:1]: What's up with the huge diff in this file?

spaces <-> tabs change?

File: Northstar.Custom/mod.json [hunk 2]
```diff
@@ -1,362 +1,404 @@
-{
```
Comment (GeckoEidechse) [inline Northstar.Custom/mod.json:1]: Same here

**Comment by GeckoEidechse** [comment]
Also short descriptions for anyone unfamiliar with the two game modes coming across this:

- **One in the Chamber:** One Shot, One Kill. Get another bullet in your clip by scoring a kill.
- **The Hidden:** One player is invisible. The others hunt them.

File: Northstar.Custom/keyvalues/playlists_v2.txt (hunk 1 above)
Comment (x3Karma) [reply Northstar.Custom/keyvalues/playlists_v2.txt:1]: i just uploaded my own files

**Comment by x3Karma** [comment]
have to upstream first

**Pull request closed without merging**